cmake/TestImpactFramework/LYTestImpactFramework.cmake

```diff
@@ -439,6 +441,45 @@ function(ly_test_impact_write_config_file CONFIG_TEMPLATE_FILE BIN_DIR)
     message(DEBUG "Test impact framework post steps complete")
 endfunction()
 
+function(ly_test_impact_write_pytest_file)
+    set(config_array "")
+    foreach(config_type ${LY_CONFIGURATION_TYPES})
+        if(NOT "${config_type}" STREQUAL "release")
+            list(APPEND config_array "\"${config_type}\":\"${LY_TEST_IMPACT_WORKING_DIR}/${config_type}/Persistent/tiaf.json\"")
+        endif()
+    endforeach()
+    string(REPLACE ";" "," CONFIG_PATHS "${config_array}")
+
+    set(bin_array "")
+    foreach(config_type ${LY_CONFIGURATION_TYPES})
+        if(NOT "${config_type}" STREQUAL "release")
+            list(APPEND bin_array "\"${config_type}\":\"${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/${config_type}/tiaf.exe\"")
+        endif()
+    endforeach()
+    string(REPLACE ";" "," BIN_PATHS "${bin_array}")
+
+    set(report_arr "")
+    foreach(config_type ${LY_CONFIGURATION_TYPES})
+        if(NOT "${config_type}" STREQUAL "release")
+            list(APPEND report_arr "\"${config_type}\":\"${LY_TEST_IMPACT_WORKING_DIR}/${config_type}/Temp\"")
+        endif()
+    endforeach()
+    string(REPLACE ";" "," REPORT_PATHS "${report_arr}")
+
+    set(BUILD_DIR ${CMAKE_BINARY_DIR})
+
+    set(ROOT_DIR ${LY_ROOT_FOLDER})
+
+    ly_file_read("cmake/TestImpactFramework/PyTestTestData.in" test_file)
```

Comment:
Suggestion: `PyTestTestData.in` could be renamed something like `TestImpactPyTestConfigurationData.in`.